src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -411,6 +437,13 @@ public CodeFixProvider GetSuppressionFixer(string language, ImmutableArray<Diagn
             return false;
         }
 
+        private bool IsInteractiveCodeFixProvider(CodeFixProvider provider)
+        {
+            // TODO (https://github.com/dotnet/roslyn/issues/4932): Don't restrict CodeFixes in Interactive
+            return provider is AddImport.AbstractAddImportCodeFixProvider ||
```

Comment:
I don't like this. Why don't we just update our existing fixers to opt-out of Interactive submissions? This otherwise leads us to a situation in Update 1 where people will ask us how they make their analyzers run in Interactive, and we won't have a useful response.